src/npdfhir/tests.py

```diff
@@ -16,7 +16,7 @@ def setup_databases(self, **kwargs):
 
         # Apply unmanaged tables
         with connection.cursor() as cursor:
-            cursor.execute(open("../db/sql/schemas/npd.sql").read())
+            cursor.execute(open("../db/sql/schemas/V1__initial_npd_schema.sql").read())
```

Comment:
I don't know yet how the python e2e tests interact with Postgres running in a docker container, talking through this now with @IsaacMilarky